app/jobs/send_patient_otp_sms_job.rb

```diff
@@ -1,14 +1,17 @@
 class SendPatientOtpSmsJob < ApplicationJob
   queue_as :default
-  self.queue_adapter = :sidekiq
```

Comment:
Have we configured Sidekiq as the default adapter for `ApplicationJob`?

Reply:
Good catch; fixed.

Our jobs are pretty inconsistent and I wanted to separate that cleanup in a different PR.